include/range/v3/utility/concepts.hpp

```diff
@@ -413,9 +413,9 @@ namespace ranges
                 template<typename T>
                 auto requires_(T&&) -> decltype(
                     concepts::valid_expr(
-                        concepts::model_of<Assignable, T, T&>(),
-                        concepts::model_of<Assignable, T, const T&>(),
-                        concepts::model_of<Assignable, T, const T&&>()
+                        concepts::model_of<Assignable, T&, T&>(),
+                        concepts::model_of<Assignable, T&, const T&>(),
+                        concepts::model_of<Assignable, T&, const T&&>()
```

Comment:
For future reference, I put const after the type, like `T const &`. It's not worth changing, though.